portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -33,9 +33,51 @@ definition {
 	@ignore = "true"
```

Comment:
@AfonsoHSantos the point of this testcase is to make sure that individual usage _does not increase_ when an anonymous individual does some activities.

An anonymous individual is someone who views a tracked site in DXP but _never logs in_.  As soon as they login, they will be converted to a _known individual_.

So, to test this, the following is required:
1. A DXP is connected to AC with contacts + analytics enabled
2. A user views a page in DXP without ever logging in.

So, in the rest of the test case, you will want to check that the user is viewing the site without logging in so that the activity is tracked as _anonymous_.

cc @alexcandidos

Reply:
Hey @christianstokes , thanks for your contribution to the discussion.

Today, the AC is taking a long time (_bigger than the test execution duration_) to do this sync after logging in, converting the _anonymous_ user to a _known_ user.  Do you know if it was supposed to be immediately synced /pushed to AC as soon as the login happens? Or if it was in this way before? and now the behavior seems to be a regression?

This is why in some tests we are using the macro ``ACDXPSettings.syncNewContactData`` to force this sync without having to wait for nanites to run doing it on their time.

So, I think the way how the code is implemented is okay.

cc: @r4ddek